Services/Init/classes/class.ilErrorHandling.php

```diff
@@ -9,12 +10,36 @@
 *
 * @author	Stefan Meyer <meyer@leifos.com>
 * @author	Sascha Hofmann <shofmann@databay.de>
+* @author	Richard Klees <richard.klees@concepts-and-training.de>
 * @version	$Id$
 * @extends PEAR
 * @todo		when an error occured and clicking the back button to return to previous page the referer-var in session is deleted -> server error
+* @todo		This class is a candidate for a singleton. initHandlers could only be called once per process anyways, as it checks for static $handlers_registered.
 */
 include_once 'PEAR.php';
 
+// TODO: This would clearly benefit from Autoloading...
+require_once("./Services/Exceptions/lib/Whoops/Run.php");
+require_once("./Services/Exceptions/lib/Whoops/Handler/HandlerInterface.php");
+require_once("./Services/Exceptions/lib/Whoops/Handler/Handler.php");
+require_once("./Services/Exceptions/lib/Whoops/Handler/CallbackHandler.php");
+require_once("./Services/Exceptions/lib/Whoops/Handler/PrettyPageHandler.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/Inspector.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/ErrorException.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/FrameCollection.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/Frame.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/Inspector.php");
+require_once("./Services/Exceptions/lib/Whoops/Exception/Formatter.php");
+require_once("./Services/Exceptions/lib/Whoops/Util/TemplateHelper.php");
+require_once("./Services/Exceptions/lib/Whoops/Util/Misc.php");
+
+require_once("Services/Exceptions/classes/class.ilDelegatingHandler.php");
+
+use Whoops\Run;
+use Whoops\Handler\PrettyPageHandler;
+use Whoops\Handler\CallbackHandler;
+use Whoops\Exception\Inspector;
+
 class ilErrorHandling extends PEAR
```

Comment:
thanks for the info, I wasn't clear about the implementation. In that case the extension makes sense. Since the feature was discussed ah the JF, I think this could be merged into trunk? Or what was the agreed procedure?